frontend/src/components/program/FormScheduleEntryItem.vue

```diff
@@ -5,7 +5,8 @@
         <e-date-picker
           v-model="localScheduleEntry.start"
           value-format="YYYY-MM-DDTHH:mm:ssZ"
-          :name="$tc('components.program.formScheduleEntryItem.start')"
+          :label="$tc('components.program.formScheduleEntryItem.start')"
+          name="startDate"
```

Comment:
minor:
```suggestion
          name="startDate"
          :label="$tc('components.program.formScheduleEntryItem.start')"
```


I would keep the order in this PR, and specially in this file, consistent